### PR TITLE
Add app server span tags

### PR DESCRIPTION
**Pull request by brianashby-sfx**
This PR provides information about span tags the instrumentation adds to `SERVER` spans for app servers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,15 @@ A simple wrapper for the Jaeger exporter of [opentelemetry-java](https://github.
 ### More options
 For more options see [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation#configuration-parameters-subject-to-change)
 
+## Span tags the Splunk distribution automatically creates
+
+This Splunk distribution automatically adds context to `SERVER` spans for application servers with span tags. These are the span tags:
```
Comment (vovencij) [inline README.md:120]: maybe `adds context to ... spans for known application servers using span tags`?

**Review by vovencij — APPROVED** [review]

**Comment by vovencij** [comment]
Argh, collector broke stuff for us. Ago's PR (https://github.com/signalfx/splunk-otel-java/pull/87) has the fix. Will have to wait until he merges.

**Comment by iNikem** [comment]
@brianashby-sfx Can you please rebase this PR?

**Comment by vovencij** [comment]
@brianashby-sfx please rebase

**Pull request merged**